SPECS/mariner-repos/mariner-official-base.repo

```diff
@@ -7,3 +7,23 @@ repo_gpgcheck=1
 enabled=1
 skip_if_unavailable=True
 sslverify=1
+
+[mariner-official-base-debuginfo]
+name=CBL-Mariner Official Base $releasever $basearch Debuginfo
+baseurl=https://packages.microsoft.com/cbl-mariner/$releasever/prod/base/debuginfo/$basearch
+gpgkey=file:///etc/pki/rpm-gpg/MICROSOFT-RPM-GPG-KEY file:///etc/pki/rpm-gpg/MICROSOFT-METADATA-GPG-KEY
+gpgcheck=1
+repo_gpgcheck=1
+enabled=0
```

Comment:
I like the idea but this will also require a change in our tooling for how to include debuginfo packages in the builds. So far we've been using the `REPO_LIST` argument with the assumption that there's a separate .repo file for the debuginfo packages and that the repo is enabled by default. Not sure, how to get that granularity right now (debuginfo vs "regular" packages) but I doubt someone would want only the debug packages and not the "regular" ones, so we could for instance add a new `ENABLE_DEBUGINGO_PACKAGES` argument, which would then enable all debuginfo repos inside the .repo files used during the build.

Reply:
And if you're curious who's using debuginfo packages during builds: I do, just started. ;) Building livepatches requires `kernel-debuginfo`. I'm not sure, if anyone else is, though, but we already said we support that during the Mariner conference.

Reply:
Hmm..yeah..good catch Pawel.